cui.cpp

```diff
@@ -215,6 +216,18 @@ void exit_ui ()
 
 void ui_tick ()
 {
+	fd_set fds;
+	
+	int maxfd = 0; // stdin
+
+	FD_ZERO(&fds);
+	FD_SET(0, &fds);
+	int status = select(maxfd + 1, &fds, 0, 0, 0);
+	
+	if(status == -1){
+	  std::cerr << "select error: " << strerror(errno) << std::endl;
```

Comment:
This is triggered when select() is interrupted when resizing the terminal nethogs is running in